app.js

```diff
@@ -15,5 +15,39 @@ function closeNav() {
     navMenu.style.height = "50px"
 }
 
+let inputName = document.querySelector(".input-name")
+let inputEmail = document.querySelector(".input-email")
+let inputSubject = document.querySelector(".input-subject")
+let inputMessage = document.querySelector(".input-message")
+let button = document.querySelector(".field-button")
+
+let info = {
+    name: "",
+    email: "",
+    subject: "",
+    message: ""
+}
+
+let arrayInfo = []
+
+button.onclick = () => {
+    info.name = inputName.value
+    info.email = inputEmail.value
+    info.subject = inputSubject.value
+    info.message = inputMessage.value
+    if (info.name != "" && info.email != "" && info.subject != "" && info.message != "") {
```

Comment:
I think will be better: `if (info.name && info.email && info.subject && info.message)`